### Laelaps laelaps_control/IsAlarmed service
#
# Test if robot is in an alarmed state.
#

---
bool is_alarmed   # robot is [not] alarmed
